src/main/java/org/pesmypetcare/webservice/dao/medalmanager/MedalDaoImpl.java

```diff
@@ -82,4 +88,21 @@ private void initializeWithCollectionPath() {
         batch = dbCol.batch();
         path = Path.ofCollection(Collections.medals);
     }
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5630992)